web/app/js/components/TapEventTable.jsx

```diff
@@ -0,0 +1,136 @@
+import _ from 'lodash';
+import BaseTable from './BaseTable.jsx';
+import { publicAddressToString } from './util/Utils.js';
+import React from 'react';
+import { Col, Icon, Row } from 'antd';
+
+let tapColumns = [
+  {
+    title: "ID",
+    dataIndex: "base.id"
+  },
+  {
+    title: "Direction",
+    dataIndex: "base.proxyDirection"
+  },
+  {
+    title: "Source",
+    dataIndex: "base.source",
+    render: d => !d ? null : <span>{publicAddressToString(_.get(d, "ip.ipv4"), d.port)}</span>
+  },
+  {
+    title: "Destination",
+    dataIndex: "base.destination",
+    render: d => !d ? null : <span>{publicAddressToString(_.get(d, "ip.ipv4"), d.port)}</span>
+  },
+  {
+    title: "TLS",
+    dataIndex: "base.tls"
+  },
+  {
+    title: "Request Init",
+    children: [
+      {
+        title: "Authority",
+        key: "authority",
+        dataIndex: "req.http.requestInit",
+        render: d => !d ? <Icon type="loading" /> : d.authority
+      },
+      {
+        title: "Path",
+        key: "path",
+        dataIndex: "req.http.requestInit",
+        render: d => !d ? <Icon type="loading" /> : d.path
+      },
+      {
+        title: "Scheme",
+        key: "scheme",
+        dataIndex: "req.http.requestInit",
+        render: d => !d ? <Icon type="loading" /> : _.get(d, "scheme.registered")
+      },
+      {
+        title: "Method",
+        key: "method",
+        dataIndex: "req.http.requestInit",
+        render: d => !d ? <Icon type="loading" /> : _.get(d, "method.registered")
+      }
+    ]
+  },
+  {
+    title: "Response Init",
+    children: [
+      {
+        title: "HTTP status",
+        key: "http-status",
+        dataIndex: "rsp.http.responseInit",
+        render: d => !d ? <Icon type="loading" /> : d.httpStatus
+      },
+      {
+        title: "Latency",
+        key: "rsp-latency",
+        dataIndex: "rsp.http.responseInit",
+        render: d => !d ? <Icon type="loading" /> : d.sinceRequestInit
```

Comment:
Looks like this duration is in seconds? We might want to consider multiplying by 1000 and then calling `formatLatency` on it. Same goes for the `sinceResponseInit` field.